<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwiftGuiStd</class>
 <widget class="QMainWindow" name="SwiftGuiStd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>swift GUI</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks</set>
  </property>
  <widget class="QWidget" name="wi_CentralWidgetOutside">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="vl_CentralWidgetOutside">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="BlackGui::COverlayMessagesFrame" name="fr_CentralFrameInside">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="vl_CentralFrameInside">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="sw_MainMiddle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pg_MainInfoArea">
          <layout class="QVBoxLayout" name="vl_MainInfoArea">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="BlackGui::Components::CMainInfoAreaComponent" name="comp_MainInfoArea" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pg_LoginConfirmation">
          <layout class="QVBoxLayout" name="vl_LoginConfirmation">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="BlackGui::Components::CLoginComponent" name="comp_Login" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pg_Internals">
          <layout class="QVBoxLayout" name="vl_Internals">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="BlackGui::Components::CInternalsComponent" name="comp_Internals" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="BlackGui::Components::CMainKeypadAreaComponent" name="comp_MainKeypadArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mb_MainMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menu_Test">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Test</string>
    </property>
    <widget class="QMenu" name="menu_PlanePositions">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Set plane positions </string>
     </property>
     <addaction name="menu_TestLocationsEDRY"/>
     <addaction name="menu_TestLocationsEDDF"/>
     <addaction name="menu_TestLocationsEDDM"/>
     <addaction name="menu_TestLocationsEDNX"/>
     <addaction name="menu_TestLocationsLOWW"/>
    </widget>
    <widget class="QMenu" name="menu_Internals">
     <property name="title">
      <string>Internals</string>
     </property>
     <addaction name="menu_InternalsPage"/>
    </widget>
    <addaction name="menu_Internals"/>
    <addaction name="menu_PlanePositions"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_InfoAreas">
    <property name="title">
     <string>&amp;Info areas</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="menu_WindowFont"/>
    <addaction name="menu_WindowToggleNavigator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="menu_MovingMap"/>
    <addaction name="menu_AutoPublish"/>
    <addaction name="menu_ToggleIncognito"/>
    <addaction name="menu_ModelBrowser"/>
    <addaction name="menu_AfvMap"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_InfoAreas"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
   <addaction name="menu_Test"/>
  </widget>
  <widget class="QStatusBar" name="sb_MainStatusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="BlackGui::CDockWidgetInfoBar" name="dw_InfoBarStatus">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Info status bar</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="qw_InfoBarStatusInner">
    <layout class="QVBoxLayout" name="vl_InfoBarStatusOuter">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignLeft|Qt::AlignTop">
      <widget class="BlackGui::Components::CInfoBarStatusComponent" name="comp_InfoBarStatus" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="menu_TestLocationsEDDF">
   <property name="text">
    <string>Position EDDF (Frankfurt, GER)</string>
   </property>
  </action>
  <action name="menu_TestLocationsEDDM">
   <property name="text">
    <string>Position EDDM (Munich, GER)</string>
   </property>
  </action>
  <action name="menu_TestLocationsEDRY">
   <property name="text">
    <string>Position EDRY (Speyer, GER)</string>
   </property>
  </action>
  <action name="menu_ReloadSettings">
   <property name="text">
    <string>Reload settings</string>
   </property>
  </action>
  <action name="menu_TestLocationsEDNX">
   <property name="text">
    <string>Position EDNX (Schleißheim, GER)</string>
   </property>
  </action>
  <action name="menu_FileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menu_FileSettingsDirectory">
   <property name="text">
    <string>Settings directory</string>
   </property>
  </action>
  <action name="menu_FileResetSettings">
   <property name="text">
    <string>Reset settings</string>
   </property>
  </action>
  <action name="menu_FileReloadStyleSheets">
   <property name="text">
    <string>Reload style sheets</string>
   </property>
  </action>
  <action name="menu_WindowFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="menu_DebugMetaTypes">
   <property name="text">
    <string>Meta types (to console)</string>
   </property>
  </action>
  <action name="menu_TestLocationsLOWW">
   <property name="text">
    <string>Position LOWW (Vienna, AUT)</string>
   </property>
  </action>
  <action name="menu_WindowMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W, Ctrl+M</string>
   </property>
  </action>
  <action name="menu_WindowToggleOnTop">
   <property name="text">
    <string>Toggle stay on top</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W, Ctrl+T</string>
   </property>
  </action>
  <action name="menu_WindowToggleNavigator">
   <property name="text">
    <string>Navigator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W, Ctrl+N</string>
   </property>
  </action>
  <action name="menu_NavigatorHorizontal">
   <property name="text">
    <string>Horizontal navigator</string>
   </property>
  </action>
  <action name="menu_InternalsMetatypes">
   <property name="text">
    <string>Metatypes</string>
   </property>
  </action>
  <action name="menu_InternalsEnvVars">
   <property name="text">
    <string>Env.variables</string>
   </property>
  </action>
  <action name="menu_InternalsSetup">
   <property name="text">
    <string>Setup</string>
   </property>
  </action>
  <action name="menu_InternalsCompileInfo">
   <property name="text">
    <string>Compile info</string>
   </property>
  </action>
  <action name="menu_InternalsPage">
   <property name="text">
    <string>Internals page</string>
   </property>
  </action>
  <action name="menu_InternalsDeleteCachedFiles">
   <property name="text">
    <string>Delete cached files</string>
   </property>
  </action>
  <action name="menu_InternalsDisplayCachedFiles">
   <property name="text">
    <string>Display cached files</string>
   </property>
  </action>
  <action name="menu_SettingsDirectory">
   <property name="text">
    <string>Directory</string>
   </property>
  </action>
  <action name="menu_SettingsFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="menu_SettingsReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="menu_CacheDirectory">
   <property name="text">
    <string>Directory</string>
   </property>
  </action>
  <action name="menu_CacheFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="menu_CacheReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="menu_MovingMap">
   <property name="text">
    <string>Moving map</string>
   </property>
  </action>
  <action name="menu_AutoPublish">
   <property name="icon">
    <iconset resource="../blackmisc/blackmisc.qrc">
     <normaloff>:/diagona/icons/diagona/icons/paper-plane--plus.png</normaloff>:/diagona/icons/diagona/icons/paper-plane--plus.png</iconset>
   </property>
   <property name="text">
    <string>Auto-publish</string>
   </property>
  </action>
  <action name="menu_ToggleIncognito">
   <property name="text">
    <string>Toggle incognito mode</string>
   </property>
  </action>
  <action name="menu_ModelBrowser">
   <property name="text">
    <string>Model browser</string>
   </property>
  </action>
  <action name="menu_AfvMap">
   <property name="text">
    <string>AFV map</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BlackGui::COverlayMessagesFrame</class>
   <extends>QFrame</extends>
   <header>blackgui/overlaymessagesframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CInfoBarStatusComponent</class>
   <extends>QWidget</extends>
   <header>blackgui/components/infobarstatuscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CMainInfoAreaComponent</class>
   <extends>QWidget</extends>
   <header>blackgui/components/maininfoareacomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CDockWidgetInfoBar</class>
   <extends>QDockWidget</extends>
   <header>blackgui/dockwidgetinfobar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CMainKeypadAreaComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/mainkeypadareacomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CLoginComponent</class>
   <extends>QWidget</extends>
   <header>blackgui/components/logincomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CInternalsComponent</class>
   <extends>QWidget</extends>
   <header>blackgui/components/internalscomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../blackmisc/blackmisc.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>setMainPage()</slot>
  <slot>toggleNetworkConnection()</slot>
  <slot>menuClicked()</slot>
  <slot>networkServerSelected(QModelIndex)</slot>
  <slot>alterTrafficServer()</slot>
  <slot>reloadAtcStationsBooked()</slot>
  <slot>onlineAtcStationSelected(QModelIndex)</slot>
  <slot>commandEntered()</slot>
  <slot>getMetar()</slot>
  <slot>cockpitValuesChanged()</slot>
  <slot>audioDeviceSelected(int)</slot>
  <slot>voiceRoomOverride()</slot>
 </slots>
</ui>
